src/IconFocusableTagHelper.cs

```diff
@@ -1,7 +1,7 @@
-﻿namespace Tailwind.Heroicons;
+namespace Tailwind.Heroicons;
 
 /// <summary>
-/// Tag helper that sets <code>focusable="false"</code> on <see cref="IconTagHelper"/> instances.
+/// Tag helper that sets <c>focusable</c> to <c>false</c> on <see cref="IconTagHelper"/> instances.
```

Comment:
The old way broke the vscode tooltip and everything inside the `<code>` tag was excluded. The new way isn't much better but at least all of the text shows now.

<img width="413" alt="image" src="https://user-images.githubusercontent.com/831974/194743530-2c2dcab3-93d8-4075-bd3d-6902380f72fa.png">